easybuild/easyconfigs/h/h5py/h5py-2.5.0-foss-2016a-Python-3.5.1-HDF5-1.8.16.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "PythonPackage"
+
+name = 'h5py'
+version = '2.5.0'
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+hdf5ver = '1.8.16'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s' % hdf5ver
+
+# to really use mpi enabled hdf5 we now seem to need a configure step
+prebuildopts = ' python setup.py configure --mpi --hdf5=$EBROOTHDF5 && '
+
+patches = ['h5py-2.5.0-foss-2016a-Python-3.5.1-HDF5-1.8.16.patch']
```

Comment:
don't hardcode the `h5py` version in the patch file name, please use `%(version)s`

also: move this up below `sources`